build/dind-test/base/Makefile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY:	build push
+
+# TAG is the version to build and push to.
+PREFIX = gcr.io/qlee_dev
+NAME = dind-test-base
+ARCH = amd64
+TAG = 0.1
+
+IMAGE_NO_TAG = ${PREFIX}/${NAME}-${ARCH}
+IMAGE = ${IMAGE_NO_TAG}:$(TAG)
```

Comment:
`IMAGE = gcr.io/qlee_dev/dind-test-base-amd64:0.1` seems a lot easier to read here